gunicorn/arbiter.py

```diff
@@ -126,28 +125,23 @@ def start(self):
         """
         self.log.info("Starting gunicorn %s", __version__)
 
+        if 'GUNICORN_PID' in os.environ:
+            self.master_pid = int(os.environ.get('GUNICORN_PID'))
+            self.proc_name = self.proc_name + ".2"
+            self.master_name = "Master.2"
```

Comment:
right now it can't happen since the string is fixed. When don't do a `+` but set the master name.